lib/tapioca/helpers/config_helper.rb

```diff
@@ -0,0 +1,85 @@
+# typed: strict
+# frozen_string_literal: true
+
+require "yaml"
+
+module Tapioca
+  module ConfigHelper
+    extend T::Sig
+
+    sig { returns(String) }
+    attr_reader :command_name
+
+    sig { params(args: T.untyped, local_options: T.untyped, config: T.untyped).void }
+    def initialize(args = [], local_options = {}, config = {})
+      # Store current command
+      command = config[:current_command]
+      command_options = config[:command_options]
+      @command_name = T.let(command.name, String)
+      @merged_options = T.let(nil, T.nilable(Thor::CoreExt::HashWithIndifferentAccess))
+      @defaults = T.let(nil, T.nilable(Thor::CoreExt::HashWithIndifferentAccess))
```

Comment:
Can't we initialize those with a `Thor::CoreExt::HashWithIndifferentAccess.new` directly? Why do we need a `nil`?

Reply:
I did that for `@defaults` but `@merged_options` is being used for memoization, so I didn't change that.